pkg/config/analysis/analyzers/virtualservice/destinationrules.go

```diff
@@ -116,10 +120,18 @@ func initDestHostsAndSubsets(ctx analysis.Context) map[hostAndSubset]bool {
 	ctx.ForEach(collections.IstioNetworkingV1Alpha3Destinationrules.Name(), func(r *resource.Instance) bool {
 		dr := r.Message.(*v1alpha3.DestinationRule)
 		drNamespace := r.Metadata.FullName.Namespace
-
+		drHost := dr.GetHost()
+		if drNamespace == util.IstioSystemNamespace && drHost != "" && host.IsShortNameForHost(drHost) {
```

Comment:
Using IstioSystem is not accurate, that is the default but not only option. Root namespace is the right thing to look for.

Reply:
Yes, you are correct, I will change this to root namespace.

Reply:
BTW, there should be no IstioSystemNamespace any more, because this warning information is unnecessary.